kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -229,7 +229,7 @@
   }
 
   .username {
-    max-width: 200px;
+    max-width: 50px;
```

Comment:
This is too small. It truncates the username even when the window is full sized.